.github/workflows/maven-ci.yml

```diff
@@ -0,0 +1,108 @@
+name: OTP CI Build
+
+# TODO: Perhaps reinstate, but doesn't allow for build/test of any branch.
+# On [push, pull_request] causes double-builds when creating PRs.
+# But triggering on push only will miss pull requests from outside authors.
+# The push event's ref is the name of the pushed branch;
+# The pull_request event's branch name is the merge target branch.
+#on:
+#  push:
+#    branches:
+#      - master
+#      - dev-1.x
+#      - dev-2.x
+#      - ibi-dev
+#  pull_request:
+#    branches:
+#      - master
+#      - dev-1.x
+#      - dev-2.x
+#      - ibi-dev
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      JAVA_OPTS: -Xmx2g
+      # TODO Remove if not required.
+      OSSRH_JIRA_USERNAME: ${{ secrets.OSSRH_JIRA_USERNAME }}
+      OSSRH_JIRA_PASSWORD: ${{ secrets.OSSRH_JIRA_PASSWORD }}
+      GPG_KEY_NAME: ${{ secrets.GPG_KEY_NAME }}
+      GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
```

Comment:
Yea, I think it's unneeded (only for the main repo).

Reply:
Removed.